articles/autoscaling-in-aws/index.md

```diff
@@ -0,0 +1,98 @@
+
+After an AMI has been created, the configuration should be launched. At the launch configuration, the right type of instance should be chosen based on need. An AWS autoscaling group should be created after a configuration has been launched. This group will be used in the launching of new instances. 
+
+![AWS Autoscaling](/engineering-education/autoscaling-in-amazon-web-services/aws-autoscaling.png)
+[Image Source: Jayendra](https://jayendrapatil.com/wp-content/uploads/2016/03/AWS-Auto-Scaling-Configurations.png)
+
+When there is a heavy traffic, the AWS autoscaling service will initiate the addition of new EC2 instances with identical configuration to the server. This happens with the aid of AMI. When there is relatively less traffic, the excess number of instances will be removed.
+
+When new instances are launched, the load balancer will divide traffic among them. The system load is used as the basis for the division of traffic. The ratio of division depends on the type of load balancer employed. In the case of a classic load balancer, the traffic will be divided equally. In application balancer, some rules will be used to guide the division of traffic.
+
+### Benefits of AWS Autoscaling
+AWS autoscaling provides the following benefits to users;
+* AWS autoscaling enables users to minimize costs. This is because total usage is the basis for the price charged, rather than capacity. When the load is low, autoscaling can enable some unutilized servers to be asleep, which reduces electricity costs. 
+* It can help in protecting against application, hardware, and network failures. This is because unhealthy instances can be replaced by the AWS autoscaling service. 
+* It enhances smart scaling decisions. This autoscaling service enables users to create resourceful scaling plans in which resources can be changed automatically to respond to any shift in demand.
+* It enhances the optimal performance of applications. This is achieved because the autoscaling service monitors applications continually. Optimal performance happens even when workloads are unpredictable.
+* AWS autoscaling allows users to set up scaling fast. Target utilization levels can be set for multiple resources using a single interface.
+
+### Conclusion
+Amazon Web Services provide various tools that enhance data management, storage, development, and networking in organizations. Autoscaling in AWS is important to organizations because it helps in optimizing utilization, costs, and performance. This autoscaling service can be an ideal choice for organizations or companies that want to optimize the performance of their applications.
+
+### Resources
+
+[Linux Daddy](https://linuxdady.com/aws-autoscaling-configuration-step-by-step-for-beginners/)
+
+[Amazon](https://docs.aws.amazon.com/autoscaling/ec2/userguide/what-is-amazon-ec2-auto-scaling.html#:~:text=%20What%20is%20Amazon%20EC2%20Auto%20Scaling%3F%20,an%20AWS%20account,%20you%20can%20access...%20More%20)
+

```

Comment:
Really good article, @mbaabuones!
I have one suggestion, could you please add a section where you give a mini-tutorial on how to set up auto scaling on an instance. 
Your article provided very good insight on Autoscaling, but I felt the article would provide more value if you give some hands-on to the topic you're discussing about.